modules/caddytls/matchers.go

```diff
@@ -330,3 +339,17 @@ var (
 	_ caddyfile.Unmarshaler = (*MatchRemoteIP)(nil)
 	_ caddyfile.Unmarshaler = (*MatchServerName)(nil)
 )
+
+// PrivateRangesCIDR returns a list of private CIDR range
+// strings, which can be used as a configuration shortcut.
+// Copied from caddyhttp/ip_range.go due to an import cycle.
+func PrivateRangesCIDR() []string {
```

Comment:
You can probably unexport this func

Reply:
As stated in the comment, I copied this function from caddyhttp, where it's exported. Do you think there is a better place in the mainline code to move this function to, so that caddyhttp, caddytls and layer4 module could call it consistently?

Reply:
You can move it under `internal`